assets/src/apps/dup.tsx

```diff
@@ -17,7 +17,7 @@ import {
   MultiRotationPage,
   SimulationPage,
 } from "Components/dup/dup_screen_page";
-import { isDup } from "Util/util";
+import { isDup } from "Util/outfront";
```

Comment:
I unresolved a previous comment thread to ask about this in better detail. (Just noting here so you don't miss it.)

Reply:
Yeah, if I hadn't updated this and other references to `isDup` in the old code, I think the code would fail to compile or at least print some warnings to the console even though we don't have any screens configured to use that app bundle anymore.

The real reason it's updated though, is simply that it's easier to do so. I used the vscode refactor function (F2) to rename the function as well as automatically update all references to it. It would have been harder to go back afterward and undo the renames in the old files.

Reply:
> The real reason it's updated though, is simply that it's easier to do so. I used the vscode refactor function (F2)

Love it! That's a great reason.